content/docs/started/getting-started-k8s.md

```diff
@@ -144,11 +144,342 @@ kubectl get cm dex -n kubeflow -o jsonpath='{.data.config\.yaml}' > dex-config.y
+        {{< /highlight >}}
+        
+        </details>
+        
+        
+    2. Seed the LDAP database with new entries.
+        
+        ```bash
+        kubectl exec -it -n kubeflow ldap-0 -- bash
+        ldapadd -x -D "cn=admin,dc=arrikto,dc=com" -W
+        # Enter password "admin".
+        # Press Ctrl+D to complete after pasting the snippet below.
+        ```
+        
+        <details>
+        
+        <summary>LDAP Seed Users and Groups</summary>
+        ```ldif
+        # This already exists.
+        # If it doesn't, uncomment this.
+        #dn: dc=arrikto,dc=com
+        #objectClass: dcObject
+        #objectClass: organization
+        #o: Arrikto
+        #dc: arrikto
+        
+        dn: ou=People,dc=arrikto,dc=com
+        objectClass: organizationalUnit
+        ou: People
+        
+        dn: cn=Yannis Zarkadas,ou=People,dc=arrikto,dc=com
+        objectClass: person
+        objectClass: inetOrgPerson
+        givenName: Yannis
+        sn: Zarkadas
+        cn: Yannis Zarkadas
+        uid: yanniszark
+        mail: yanniszark@arrikto.com
+        userpassword: 12341234
+        
+        dn: cn=Vangelis Koukis,ou=People,dc=arrikto,dc=com
+        objectClass: person
+        objectClass: inetOrgPerson
+        givenName: Vangelis
+        sn: Koukis
+        cn: Vangelis Koukis
+        uid: vkoukis
+        mail: vkoukis@arrikto.com
+        userpassword: 43214321
+        
+        # Group definitions.
+        
+        dn: ou=Groups,dc=arrikto,dc=com
+        objectClass: organizationalUnit
+        ou: Groups
+        
+        dn: cn=admins,ou=Groups,dc=arrikto,dc=com
+        objectClass: groupOfNames
+        cn: admins
+        member: cn=Yannis Zarkadas,ou=People,dc=arrikto,dc=com
+        
+        dn: cn=developers,ou=Groups,dc=arrikto,dc=com
+        objectClass: groupOfNames
+        cn: developers
+        member: cn=Yannis Zarkadas,ou=People,dc=arrikto,dc=com
+        member: cn=Vangelis Koukis,ou=People,dc=arrikto,dc=com
+        ```
+        
+        </details>
+
+2. To use your LDAP/AD server with Dex, you have to edit the Dex config.To edit the ConfigMap containing the Dex config, run:
+
+    1. Get the current Dex config from the corresponding Config Map.
+        
+        {{< highlight bash >}}
+        kubectl get configmap -n kubeflow -o jsonpath='{.data.config\.yaml}' > dex-config.yaml
+        {{< /highlight >}}
+
+    1. Add the LDAP-specific options. Here is an example to help you out. It is configured to work with the example LDAP Server you set up previously.
+        
+        <details>
+        
+        <summary>Dex LDAP Config Section</summary>
+        {{< highlight yaml >}}
+        connectors:
+        - type: ldap
+          # Required field for connector id.
+          id: ldap
+          # Required field for connector name.
+          name: LDAP
+          config:
+            # Host and optional port of the LDAP server in the form "host:port".
```

Comment:
These comments within the config file are great. If possible, it would be good to have similar comments in the LDAP setup files above, explaining the use of usernames, etc. See my previous comment about providing example names.